doc/nrf/ug_nrf91_features.rst

```diff
@@ -65,30 +65,28 @@ See :ref:`ug_bootloader` for more information.
 A bootloader chain is optional.
 Not all of the nRF9160 samples include a secure bootloader chain, but the ones that do use the :ref:`bootloader` sample and :doc:`mcuboot:index-ncs`.
 
+Trusted Firmware-M (TF-M)
+-------------------------
+
+Trusted Firmware-M provides a configurable set of software components to create a Trusted Execution Environment.
+It is replacing Secure Partition Manager in being the default solution used by most |NCS| applications and samples.
```

Comment:
```suggestion
It is replacing Secure Partition Manager as the default solution used by most |NCS| applications and samples.
```
Maybe also use "has replaced" so we don't need to change the tense in future releases?